### DISCUSSION TITLE
Varified student education account, but github copilot sub is ended

### DISCUSSION THREAD

**Discussion by Mai0313** [community/community#40025] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I know it may be a duplicate topic that some people are facing this, but my issue is a little bit different that my payment page shows this (pic1) which means it's still activated, but on copilot page and VS code, it shows me my sub has ended (pic2&3)

![image](https://user-images.githubusercontent.com/21039366/203871865-8c014dad-13d0-4ee0-b877-234743f1bbac.png)
![image](https://user-images.githubusercontent.com/21039366/203872049-b554e682-2255-4a67-b508-5ed926e5281a.png)
![image](https://user-images.githubusercontent.com/21039366/203871982-43fd5eb9-9e4e-4ac9-a44c-d06867bf5fad.png)

Following list is what I have done so far by searching from github to try to solve this issue
1. Login github on VS code again
2. restart VS code
3. re-install copilot extension
4. change back to last version for copilot extension

Nothing works to me, I am guessing github copilot server is down? or how can I fix this issue
![image](https://user-images.githubusercontent.com/21039366/203872462-44856926-2754-4784-92cb-a5ddc4149a5c.png)
![image](https://user-images.githubusercontent.com/21039366/203872719-fe8e571c-da92-498b-b172-2d990b38032f.png)

**Comment by hjx159** [comment]
I have the same problem as you, but I have no choice╮(╯▽╰)╭

**Comment by angel-star** [comment]
I have the same problem as you, but I have no choice╮(╯▽╰)╭

**Comment by trainma** [comment]
I have the same problem as you, but I have no choice╮(╯▽╰)╭

**Comment by Pascal1132** [comment]
I have the same problem as you, but I have no choice╮(╯▽╰)╭

**Comment by kevineduardo** [comment]
They are having issues. https://www.githubstatus.com/

**Comment by jfullstackdev** [comment, marked as answer]
I'm seeing a lot of these things, you can check this out since one GitHub Staff already responded there

https://github.com/orgs/community/discussions/40037
